### PR TITLE
UI tweaks, cleanup

### PR DESCRIPTION
**Pull request by DarksightKellar**
Addresses feedback comments in #2601

Screenshot 2024-12-07 at 17 57 23
Screenshot 2024-12-07 at 17 58 23
Screenshot 2024-12-07 at 17 58 48
Screenshot 2024-12-09 at 15 33 34

### CODE REVIEW

**Review by parkermccurley — APPROVED** [review]
Looks great thanks Kelvin!

File: src/i18n/locales/en/daoCreate.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "addNFTButton": "Import another token",
   "errorMinSigners": "Number of owners must be greater than 0.",
-  "errorLowSignerThreshold": "Threshold must be greater than 0",
+  "errorLowSignerThreshold": "Threshold must be greater than 0.",
   "errorHighSignerThreshold": "Threshold must be less than number of owners.",
   "labelSigThreshold": "Threshold",
-  "helperSigThreshold": "The number of owners required to approve a transaction on this multisig",
+  "helperSigThreshold": "The number of owners required to approve a transaction on this multisig.",
```
Comment (Da-Colon) [inline src/i18n/locales/en/daoCreate.json:7]: Ah I see you did this as well. #2619 all good.

File: src/i18n/locales/en/settings.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "modulesAndGuardsTitle": "Modules and Guards",
   "permissionsTitle": "Permissions",
   "daoSettingsGeneral": "General",
-  "daoSettingsGovernance": "Governance Parameters",
+  "daoSettingsGovernance": "Governance",
```
Comment (Da-Colon) [inline src/i18n/locales/en/settings.json:18]: and this one. but all good I covered the other copy updates in the PRD

File: src/store/roles/useRolesStore.ts [hunk 3]
```diff
@@ -119,11 +121,20 @@ const useRolesStore = create<RolesStore>()((set, get) => ({
...
+            : roleHat.payments
+          )?.map(p => ({
+            ...p,
+            asset: {
+              ...p.asset,
+              logo:
+                getValue({ cacheName: CacheKeys.TOKEN_INFO, tokenAddress: p.asset.address })
+                  ?.logoUri || '',
+            },
+          })),
```
Comment (Da-Colon) [inline src/store/roles/useRolesStore.ts:137]: would it be easier to just do this in `useHatTree` instead?

**Review by Da-Colon — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en/daoCreate.json (hunk 1 above)
Comment (DarksightKellar) [reply src/i18n/locales/en/daoCreate.json:7]: lmao

File: src/store/roles/useRolesStore.ts (hunk 3 above)
Comment (DarksightKellar) [reply src/store/roles/useRolesStore.ts]: Done-zo. Thanks for this

File: src/components/DaoCreator/formComponents/AzoriusGovernance.tsx [hunk 4]
```diff
@@ -36,11 +36,11 @@ function DayStepperInput({
     <InputGroup>
       <Flex
         flexDirection="column"
-        w="100%"
         gap="0.5rem"
+        w="250"
```
Comment (adamgall) [inline src/components/DaoCreator/formComponents/AzoriusGovernance.tsx:40]: I assume this `w` unit defaults to `px`, but can you please be explicit about it?

File: src/components/DaoCreator/formComponents/Multisig.tsx [hunk 5]
```diff
@@ -142,11 +142,12 @@ export function Multisig(props: ICreationStepProps) {
             }
             isRequired
           >
-            {/* @todo replace with stepper input */}
-            <NumberStepperInput
-              onChange={value => validateNumber(value, 'multisig.signatureThreshold')}
-              value={values.multisig.signatureThreshold}
-            />
+            <Flex w="200px">
```
Comment (adamgall) [inline src/components/DaoCreator/formComponents/Multisig.tsx:145]: Do this width (and the one in the comment above) look good / scale correctly on all screen sizes?

**Review by adamgall — APPROVED** [review]
Approving but have a couple of questions (left as individual comments above) about hardcoded width values.

File: src/components/DaoCreator/formComponents/Multisig.tsx (hunk 5 above)
Comment (DarksightKellar) [reply src/components/DaoCreator/formComponents/Multisig.tsx:145]: Yep, in that it's small enough to look good all the way down to mobile screens, and fits in on desktop views as well. Also very specific width from @nicolaus-sherrill. I did add 50px to accommodate for the "days" inline text (re: 250px above)

**Review by Da-Colon — APPROVED** [review]

**Comment by nicolaus-sherrill** [comment]
> Approving but have a couple of questions (left as individual comments above) about hardcoded width values.

@adamgall 
Don't like them either...I shared a hardcoded width example for sizing reference. I have an issue coming later today with more guidance on layout using grid columns.

**Pull request merged**